legacy/definition.go

```diff
@@ -863,12 +864,13 @@ func migrateRuleSet(lang utils.Language, r RuleSet, localization flows.Localizat
 
 		// subflow rulesets operate on the child flow status
 		router = routers.NewSwitchRouter(defaultExit, "@child.status", cases, resultName)
+		uiType = "subflow"
```

Comment:
@ericnewcomer not seeing a complete list of these so thus far just have:

* "subflow"
* "split_by_groups"
* "wait_for_response"
* "split_by_expression"

Reply:
I think that's the only ones we have implemented so far.

What else is there, I think this is close if not complete:
split_by_sms_form, split_by_contact_field, split_by_flow_result

Reply:
after discussion, we're using these names:

* wait_for_response
* subflow
* split_by_groups
* split_by_expression
* split_by_contact_field
* split_by_run_result
* split_by_run_result_delimited